### PR TITLE
Request query list

### PR DESCRIPTION
**Pull request by Zarasan**
My fix for #199

### CODE REVIEW

**Comment by WolfgangSenff** [comment]
What exactly is the "formatting" this is trying to fix? I'm a little confused by #199.

**Comment by Zarasan** [comment]
Basically instead of having a single array filled with dictionaries, I structure them in FirestoreDocuments so they don't give {booleanValue: False} and such

File: addons/godot-firebase/firestore/firestore_task.gd [hunk 1]
```diff
@@ -138,7 +138,9 @@ func _on_request_completed(result : int, response_code : int, headers : PoolStri
                 data = null
                 emit_signal("delete_document")
             Task.TASK_QUERY:
-                data = bod
+                data = []
+                for doc in bod:
```
Comment (fenix-hub) [inline addons/godot-firebase/firestore/firestore_task.gd:142]: What if the result of a query is empty?

**Review by fenix-hub — CHANGES_REQUESTED** [review]

File: addons/godot-firebase/firestore/firestore_task.gd (hunk 1 above)
Comment (Zarasan) [reply addons/godot-firebase/firestore/firestore_task.gd:142]: It should give create_time for request, I added an if check if doc.has('document') so it doesn't error out if query returns no documents

**Comment by fenix-hub** [comment]
Thanks for the changes, I'll test it later today and merge it 👍

**Pull request merged**